keras/src/ops/numpy.py

```diff
@@ -6411,17 +6411,20 @@ def compute_output_spec(self, x):
         repeats = self.repeats
         if isinstance(repeats, int):
             repeats = [repeats]
+        else:
+            repeats = list(repeats)
+
         if len(x_shape) > len(repeats):
             repeats = [1] * (len(x_shape) - len(repeats)) + repeats
         else:
             x_shape = [1] * (len(repeats) - len(x_shape)) + x_shape
 
         output_shape = []
         for x_size, repeat in zip(x_shape, repeats):
-            if x_size is None:
-                output_shape.append(None)
-            else:
+            if isinstance(repeat, int):
```

Comment:
Sorry, I was incorrect in the previous review, this should be `if isinstance(x_size, int):`